{"success": false, "error": "Traceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 544, in _run_solc\n    ret: Dict = json.loads(stdout)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\njson.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 826, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 86, in process_all\n    compilations = compile_all(target, **vars(args))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 637, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 117, in __init__\n    self._compile(**kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 548, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 154, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 276, in _get_targets_json\n    return _run_solc(\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 548, in _run_solc\n    raise InvalidCompilation(f\"Invalid solc compilation {stderr}\")\ncrytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation Error: Multiple SPDX license identifiers found in source file. Use \"AND\" or \"OR\" to combine multiple licenses. Please see https://spdx.org for more information.\n--> /Users/giacomoibba/Desktop/mainnet/0a/0a771EE99E9B106b2eC035788911F22B56A541dF_RewardHandler.sol\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/utils/EnumerableSet.sol\" not found: File not found.\n --> /Users/giacomoibba/Desktop/mainnet/0a/0a771EE99E9B106b2eC035788911F22B56A541dF_RewardHandler.sol:5:1:\n  |\n5 | import \"../utils/EnumerableSet.sol\";\n  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/utils/Address.sol\" not found: File not found.\n --> /Users/giacomoibba/Desktop/mainnet/0a/0a771EE99E9B106b2eC035788911F22B56A541dF_RewardHandler.sol:6:1:\n  |\n6 | import \"../utils/Address.sol\";\n  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/utils/Context.sol\" not found: File not found.\n --> /Users/giacomoibba/Desktop/mainnet/0a/0a771EE99E9B106b2eC035788911F22B56A541dF_RewardHandler.sol:7:1:\n  |\n7 | import \"../utils/Context.sol\";\n  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@openzeppelin/contracts/access/AccessControl.sol\" not found: File not found.\n    --> /Users/giacomoibba/Desktop/mainnet/0a/0a771EE99E9B106b2eC035788911F22B56A541dF_RewardHandler.sol:1035:1:\n     |\n1035 | import '@openzeppelin/contracts/access/AccessControl.sol';\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@openzeppelin/contracts/math/SafeMath.sol\" not found: File not found.\n    --> /Users/giacomoibba/Desktop/mainnet/0a/0a771EE99E9B106b2eC035788911F22B56A541dF_RewardHandler.sol:1036:1:\n     |\n1036 | import '@openzeppelin/contracts/math/SafeMath.sol';\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"contracts/src/investment/interfaces/IRewardHandler.sol\" not found: File not found.\n    --> /Users/giacomoibba/Desktop/mainnet/0a/0a771EE99E9B106b2eC035788911F22B56A541dF_RewardHandler.sol:1038:1:\n     |\n1038 | import 'contracts/src/investment/interfaces/IRewardHandler.sol';\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"contracts/src/token/interfaces/IERC20WowsMintable.sol\" not found: File not found.\n    --> /Users/giacomoibba/Desktop/mainnet/0a/0a771EE99E9B106b2eC035788911F22B56A541dF_RewardHandler.sol:1039:1:\n     |\n1039 | import 'contracts/src/token/interfaces/IERC20WowsMintable.sol';\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"contracts/src/utils/AddressBook.sol\" not found: File not found.\n    --> /Users/giacomoibba/Desktop/mainnet/0a/0a771EE99E9B106b2eC035788911F22B56A541dF_RewardHandler.sol:1040:1:\n     |\n1040 | import 'contracts/src/utils/AddressBook.sol';\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"contracts/src/utils/interfaces/IAddressRegistry.sol\" not found: File not found.\n    --> /Users/giacomoibba/Desktop/mainnet/0a/0a771EE99E9B106b2eC035788911F22B56A541dF_RewardHandler.sol:1041:1:\n     |\n1041 | import 'contracts/src/utils/interfaces/IAddressRegistry.sol';\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@openzeppelin/contracts/token/ERC20/IERC20.sol\" not found: File not found.\n    --> /Users/giacomoibba/Desktop/mainnet/0a/0a771EE99E9B106b2eC035788911F22B56A541dF_RewardHandler.sol:1332:1:\n     |\n1332 | import '@openzeppelin/contracts/token/ERC20/IERC20.sol';\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n\n", "results": {}}
